cli_reference/tkn_cli/images

```diff
@@ -0,0 +1 @@
+../images
```

Comment:
The symlinks might be causing the build failure. Have you tried commenting them out? Or removing them and building locally?

Reply:
I will try that, thank you!